lib/db/mem.js

```diff
@@ -169,10 +169,6 @@ module.exports = function (log, error) {
   Memory.prototype.createAccountResetToken = function (tokenId, accountResetToken) {
     tokenId = tokenId.toString('hex')
 
-    if ( accountResetTokens[tokenId] ) {
```

Comment:
This behaviour didn't match the mysql backend, which silently replaces any existing reset token with a new one.  When I upgraded the test runner, the tests started failing with this `error.duplicate`.  So AFAICT we already have test coverage for, for some reason the test was just failing to fail with the old version of the runner!